stac_model/input.py

```diff
@@ -87,7 +88,7 @@ class ValueScalingProcessingExpression(ProcessingExpression):
         "wrap-inverse-map",
     ]
     | None
-)  # noqa: E501
+)
```

Comment:
Can this be aligned with the `Optional` as in the case of the above `ValueScalingObject`?

Reply:
done